src/containers/user/signout/signout.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import { Redirect } from 'react-router-dom';
+import clearToken from '../../../helpers/clearToken';
 
-const Signout = () => (
-  <div>
-    <h1>Hello Logout</h1>
-  </div>
-);
+const Signout = () => {
+  clearToken();
+
+  return (
+    <Redirect to="/signin" />
+  );
+};
```

Comment:
If possible, you could search a way you can relate this functionality with the logout button as your feature is about. for instance, as here you are redirecting a user to a login page, consume this while the user clicks the logout button.

Reply:
We don't have UI for now. Later we shall add `<a></a>` anchor tag later.